backend/service/github/github.go

```diff
@@ -96,9 +96,22 @@ type File struct {
 	LastModifiedSHA  string
 }
 
+type DirectoryEntry struct {
```

Comment:
nit -- since the entry would either be a file entry or directory entry
```suggestion
type Entry struct {
```

Reply:
Updated!